### PR TITLE
Feature/seasalt

### PR DESCRIPTION
**Pull request by bbakernoaa**
@zmoon there is a weird compilation error that is starting to occur and I can't figure out why.  I had it compiling and then something I did broke it.  Please help with this if you can

### CODE REVIEW

**Comment by bbakernoaa** [comment]
this is the error I'm getting 

```
cmake --build build -j 
[  9%] Built target testing
[  9%] Building Fortran object src/core/CMakeFiles/CATChem_core.dir/init_mod.F90.o
[ 12%] Building Fortran object src/core/CMakeFiles/CATChem_core.dir/state_mod.F90.o
/Users/l22-n04127-res/Documents/GitHub/CATChem_new/src/core/init_mod.F90:29:11:

   29 |       use MetState_Mod
      |           1
Fatal Error: Mismatch in components of derived type ‘gridstatetype’ from ‘gridstate_mod’ at (1): expecting ‘state’, but got ‘nx’
compilation terminated.
/Users/l22-n04127-res/Documents/GitHub/CATChem_new/src/core/state_mod.F90:10:8:

   10 |    use MetState_Mod, only : MetStateType
      |        1
Fatal Error: Mismatch in components of derived type ‘gridstatetype’ from ‘gridstate_mod’ at (1): expecting ‘state’, but got ‘nx’```

File: src/process/dust/ccpr_dust_common_mod.F90 [hunk 1]
```diff
@@ -44,26 +44,26 @@ module CCPr_Dust_Common_Mod
...
-      REAL(fp), ALLOCATABLE           :: DustDensity(:)            ! Dust density            [kg/m^3]
-      REAL(fp)                        :: BetaScaleFactor           ! Beta Scaling Parameter  [1]
-      REAL(fp)                        :: AlphaScaleFactor          ! Alpha Scaling Parameter [1]
-      REAL(fp), ALLOCATABLE           :: TotalEmission             ! Total emission          [kg/m^2/s]
-      REAL(fp), ALLOCATABLE           :: EmissionPerSpecies(:)     ! Emission per species    [kg/m^2/s]
+      REAL(fp), ALLOCATABLE           :: LowerBinRadius(:)       !< Lower bin radius        [m]
+      REAL(fp), ALLOCATABLE           :: UpperBinRadius(:)       !< Upper bin radius        [m]
+      REAL(fp), ALLOCATABLE           :: EffectiveRadius(:)      !< Effective radius        [m]
+      REAL(fp), ALLOCATABLE           :: DustDensity(:)          !< Dust density            [kg/m^3]
+      REAL(fp)                        :: GammaScaleFactor        !< Gamma Scaling Parameter  [1]
```
Comment (zmoon) [inline src/process/dust/ccpr_dust_common_mod.F90]: @bbakernoaa seems like this was the issue, Beta was changed to Gamma in the structure here but not somewhere else

Comment (zmoon) [reply src/process/dust/ccpr_dust_common_mod.F90]: Well the error in CI was related to this, but the one you just shared is different.

**Comment by zmoon** [comment]
> this is the error I'm getting

Some cached build things may be the issue here. If so, removing the build dir and recreating it should solve.

**Review by zmoon — APPROVED** [review]
Gave a quick look

**Pull request merged**